pal/env/src/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export function findCanvas (): { frame: HTMLDivElement, container: HTMLDivElement, canvas: HTMLCanvasElement };
+export function loadJsFile (path: string): Promise<void>;
```

Comment:
this comes from https://github.com/cocos/cocos-engine/pull/16196/files#diff-9cbea41d2702dc9457f92742092de2d9d06374d0aa38684cdcb94f80debf17e7